CRM/Member/Form/MembershipRenewal.php

```diff
@@ -855,23 +854,12 @@ public function processMembership($contactID, $membershipTypeID, $is_test, $chan
       }
       $memParams['membership_activity_status'] = ($pending || $isPayLater) ? 'Scheduled' : 'Completed';
     }
-    //CRM-4555
-    if ($pending) {
```

Comment:
@eileenmcnaughton are you saying that this couldn't ever be true? https://github.com/civicrm/civicrm-core/blob/911365c8ca40fdc1a1d9c2b6b3cc2fa2ce0ba1f1/CRM/Member/Form/MembershipRenewal.php#L590

Reply:
@seamuslee001 yes see how https://github.com/civicrm/civicrm-core/blob/2270d872ebecf5dc299fafd76b07ea07a94b1289/CRM/Member/Form/MembershipRenewal.php#L765 results in it returning before that line?

(I think the same may be the case in the function on the BAO class that this form used to share)